internal/databases/mysql/backup_fetch_handler.go

```diff
@@ -1,16 +1,125 @@
 package mysql
 
 import (
+	"bytes"
+	"os"
+	"os/exec"
+	"path"
+	"strings"
+
 	"github.com/wal-g/storages/storage"
 	"github.com/wal-g/tracelog"
 	"github.com/wal-g/wal-g/internal"
-	"os/exec"
+	"github.com/wal-g/wal-g/utility"
+)
+
+const (
+	XtrabackupApplyLogOnly   = "--apply-log-only"
+	XtrabackupIncrementalDir = "--incremental-dir"
 )
 
 func HandleBackupFetch(folder storage.Folder, backupName string, restoreCmd *exec.Cmd, prepareCmd *exec.Cmd) {
+	var err error
+	if isXtrabackupCmd(restoreCmd) {
+		tempDeltaDir := path.Join("/tmp", "delta")
```

Comment:
`/tmp`  is typically very limited in free space, database diff may take far more space
we need an option to choose directory for backup diffs